preflight-checks/src/checks/02-check-port-and-healthcheck-route.js

```diff
@@ -0,0 +1,65 @@
+import chalk from 'chalk';
+import packageJson from '../package';
+const execa = require( 'execa' );
+const waait = require( 'waait' );
+const fetch = require( 'isomorphic-fetch' );
```

Comment:
Minor: `node-fetch` is much smaller + lightweight.

```suggestion
const fetch = require( 'node-fetch' );
```